LEGO1/mxpresenter.cpp

```diff
@@ -116,8 +117,8 @@ void MxPresenter::InitVirtual()
 // OFFSET: LEGO1 0x1000bf90
 void MxPresenter::VTable0x44(undefined4 param)
 {
-  m_unk0xc |= 1 << (unsigned char)m_unk0x8;
-  m_unk0x8 = param;
+  m_unk0xc |= 1 << (unsigned char)m_someFlag;
+  m_someFlag = param;
```

Comment:
I'm not sure "someFlag" is really any more helpful than "m_unk0x8". But since we've determined some of this behavior, we can probably actually come up with some more useful names either way.

These are interesting because it changes the value at `0x8` but first uses it to store a bit flag in `0xC`. It's almost like `0xC` is holding a cache of previously set values. And then `DoneTickle`, caches `0x8` again and sets it to 0, the name implying that something has ended. Perhaps we could call `0x8` "m_currentTickleFlag" and `0xC` "m_previousTickleFlags".

Reply:
What values get set with "param"? Is there a time that either var is set to some "empty" value?

Reply:
Look at vtable 18 to 28, they all do the same thing, caching the last value of `0x8` to `0xC` and then setting `0x8`. This does that too, but with a custom value instead of a hardcoded one.

Reply:
Here's what I get with vtable0x28:
![image](https://github.com/isledecomp/isle/assets/24401303/c8d01cf4-3dc1-4ac6-826d-c4aa6b33ba0e)

Basically, a number. Should I add to `mxtypes.h` typedefs for these?
```
#typedef unsigned char TICKLEFLAGS_SIX 6;
```